README.md

```diff
@@ -13,6 +13,19 @@ Takeaway Challenge
        ':..:'                ':..:'
 
  ```
+Marianne October 2019 Cohort -
+
+1 test failing - attempt to prevent text from sending each time rspec is run.
+Also currently am able to send texts to my own number but have no corresponding rspec tests to confirm this.
+Need to deal with case where user adds something which is not on the menu.
+
+To use:
+* Load takeaway.rb in IRB
+* takeaway = Takeaway.new
+* takeaway.start_order
+* takeaway.new_order.add("Margherita")
+* takeaway.total_cost
```

Comment:
Takeaway doesn't have a `total_cost` method; I presume you mean `takeaway.new_order.total_cost`.

Reply:
Oops yes, thanks!